### ISSUE TITLE
`~/.nerves` disappears when Docker container restarts

### ISSUE THREAD

**Issue by takasehideki** [NervesJP/nerves-devcontainer#4]
`~/.nerves` contains archives of Nerves toolchains and nerves_system_br.
It will disappear when Docker container restarts or rebuilds. As a result, archives should be always downloaded on `mix deps.get`.

**Comment by takasehideki** [comment]
Same situation occurs for `~/.ssh/id_rsa`. We can resolve this situation by creating it on current (development) directory and making symbolic link to `/root/.ssh` on the Docker container.
```
mkdir .ssh
ssh-keygen -t rsa -f .ssh/id_rsa
ln -s `pwd`/.ssh/id_rsa* ~/.ssh/
```

**Comment by takasehideki** [comment]
791c15d on #2 may fixes this issue.
`mix deps.get` at first will take a while to unzip archives to `./nerves/artifacts` due to poor performance about bindings between host and container filesystems. However I think it is acceptable compared to the usual wait when running `mix deps.get`.

**Comment by takasehideki** [comment]
> Same situation occurs for `~/.ssh/id_rsa`. We can resolve this situation by creating it on current (development) directory and making symbolic link to `/root/.ssh` on the Docker container.
> 
> ```
> mkdir .ssh
> ssh-keygen -t rsa -f .ssh/id_rsa
> ln -s `pwd`/.ssh/id_rsa* ~/.ssh/
> ```

ce64a39 on #2 creates symbolic link from `/workspaces/nerves-devcontainer/.ssh/id_rsa*` on host to `/root/.ssh` on container.
Users are required to run `ssh-keygen -t rsa -N "" -f .ssh/id_rsa` at first to generate keys.

**Comment by takasehideki** [comment]
fixed by merging #2
